cypress/private/cypress_test.bzl

```diff
@@ -1,12 +1,32 @@
 load("@aspect_rules_js//js:libs.bzl", "js_binary_lib", "js_lib_helpers")
+load("@bazel_skylib//lib:dicts.bzl", "dicts")
 
-_attrs = js_binary_lib.attrs
+_attrs = dicts.add(js_binary_lib.attrs, {
+    "browsers": attr.label_list(
```

Comment:
Guessing this is temporary until we have a browser toolchain and then the attribute can be removed?

Reply:
Bingo